.github/workflows/deploy-experimental.yml

```diff
@@ -126,6 +126,14 @@ jobs:
           serverless invoke --config ./serverless-experimental.yml --function populate_content --stage dev${PR}
           echo "url=$(cat output.log | grep -m1 'ANY -' | cut -c 9-)" >> $GITHUB_OUTPUT
           popd
+      - name: condtional output for prod
+        id: test2
```

Comment:
can we make the id more descriptive like you are doing in deploy.yml?

Reply:
It was really just a POC to show how the conditionals work in github actions.  I plan to remove it before actually pushing this.  We shouldnt have 100% pointless calls in prod